ui/src/components/easter/invitation/InvitationContainer.js

```diff
@@ -1,11 +1,163 @@
-import { Text } from "@chakra-ui/react";
+import { 
```

Comment:
<img width="577" alt="Screenshot 2022-03-13 at 16 19 23" src="https://user-images.githubusercontent.com/33679483/158051469-f9f3cf23-d460-4f5a-a158-34e17a34cff9.png">

looks like:
1. Add to calendar button is not centered when the viewport is small enough
2. Think that "Celebration text" should be inline, maybe try reducing the size of the font depending on the viewport?
reference: https://chakra-ui.com/docs/styled-system/features/responsive-styles